IDE/ARDUINO/sketches/wolfssl_client/wolfssl_client.ino

```diff
@@ -52,7 +61,23 @@ void setup() {
     return;
   }
   /* initialize wolfSSL using callback functions */
-  wolfSSL_CTX_set_verify(ctx, SSL_VERIFY_NONE, 0);
+  wolfSSL_CTX_set_verify(ctx, SSL_VERIFY_PEER, 0);
+	 int rc = wolfSSL_CTX_load_verify_buffer(ctx, ca_cert_der_2048, sizeof_ca_cert_der_2048, WOLFSSL_FILETYPE_ASN1);
+	 Serial.print("\n\n Return code of load_verify is:");
+	 Serial.println(rc);
+	 Serial.println("");
+
+	 rc = wolfSSL_CTX_use_certificate_buffer(ctx, client_cert_der_2048, sizeof_client_cert_der_2048, WOLFSSL_FILETYPE_ASN1);
+	 Serial.print("\n\n Return code of use_certificate_buffer is:");
+	 Serial.println(rc);
+	 Serial.println("");
+
+	 rc = wolfSSL_CTX_use_PrivateKey_buffer(ctx, client_key_der_2048, sizeof_client_key_der_2048, WOLFSSL_FILETYPE_ASN1);
+	 Serial.print("\n\n Return code of use_PrivateKey_buffer is:");
+	 Serial.println(rc);
+	 Serial.println("");
```

Comment:
The leading spacing on these code sections looks off when viewed in the github gui. Maybe the old code had hard tabs / spacing of 2 chars, or vice versa new code does?